public/data/vishalchauhan.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Vishal Chauhan",
   "type": "personal",
-  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies",
+  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies 
```

Comment:
```suggestion
  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix